freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -150,7 +156,7 @@ public User userAdd(String user, String firstName, String lastName) throws FreeI
                 "sn", lastName,
                 "loginshell", "/bin/bash",
                 "random", true,
-                "setattr", "krbPasswordExpiration=" + PASSWORD_EXPIRATION_DATETIME
+                "setattr", "krbPasswordExpiration=" + MAX_PASSWORD_EXPIRATION_DATETIME
```

Comment:
again, for my edification, why are we setting the password expiration when we add the user?

Reply:
We set a random password so that we can generate tgts. I think that we had to set this so that the random password isn't expired.